k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1066,6 +1077,17 @@ public void onBackPressed() {
         }
     }
 
+    private boolean draftIsNotEmpty() {
```

Comment:
The recipient fields is a valid case and I've fixed it. The signature field less so. The signature field is never going to be empty because it'll be pre-populated with the default signature.